Student-Grace-Calculation/src/GradeSchemaCalculator.java

```diff
@@ -48,9 +53,14 @@ public static void main(String[] args) {
 			
 			GradeInfo gradeinfo = new GradeInfo(grades[i], min, max);
 			
+			showResults += gradeinfo.getGrade() + 
```

Comment:
Can you add another class GradesRepository and store the created grades there temporarily until displaying it back to the user. You can use ArrayList or HashMap or any other data structure that you think fit. Later we will replace this repository class with code that will talk to actual database.